### ISSUE TITLE
Get a DOI for the package

### ISSUE THREAD

**Issue by natema** [worlddynamics/Vensim2MTK.jl#26]
We can ask a DOI on [Zenodo](https://zenodo.org/record/7093581), so that we can make it possible for people to cite the software if they use it in some project, as we did for WorldDynamics: https://github.com/worlddynamics/WorldDynamics.jl#how-to-cite-this-work .

**Comment by meyde** [comment]
so how do Zenodo works? i'm a bit confused

**Comment by natema** [comment]
I don't remember by hearth, but it shouldn't be hard to follow the online documentation: https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content .

**Comment by meyde** [comment]
i can only register repository that i own, for the one in an organisation, there is a need to authorize something in the organisation side

**Comment by natema** [comment]
I managed to give Zenodo access to the repo. As soon as the first release is out (#27), we can get a DOI.

**Comment by natema** [comment]
Zenodo already generated the DOI for the project: https://zenodo.org/account/settings/github/repository/worlddynamics/Vensim2MTK.jl . 
Feel free to do a PR adding the badge and the how-to cite on the README.

**Comment by meyde** [comment]
i don't have the permission to read this link.

**Comment by meyde** [comment]
i'm writing the how-to-cite by looking at the examples written in the other modules in worlddynamics, and i have a question: 
in @software{pierluigi_crescenzi_2022_7093581, ....  
Should i change this name to yours? and with what id to follow?

**Comment by natema** [comment]
The id is arbitrary, you can make it up, e.g. `vensim2mtk_2023`.

**Comment by meyde** [comment]
alright
